content/references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/can_delete.md

```diff
@@ -47,7 +47,7 @@ The request includes the following optional parameter:
 
 | `Field`      | Type              | Description                               |
 |:-------------|:------------------|:------------------------------------------|
-| `can_delete` | String or Integer | The maximum ledger to allow to be deleted. For `ledger_index` or `ledger_hash`, see [Specifying Ledgers][]. `never` sets the value to 0, and effectively disables online deletion until another `can_delete` is appropriately called.  `always` sets the value to the maximum possible ledger (4294967295), and online deletion occurs as of each configured `online_delete` interval. `now` triggers online deletion at the next validated ledger that meets or exceeds the configured `online_delete` interval, but no further. |
+| `can_delete` | String or Integer | The [Ledger Index][] of the maximum ledger version to allow to be deleted. The special case `never` disables online deletion. The special case `always` enables automatic online deletion as if advisory deletion was disabled. The special case `now` allows online deletion one time at the next validated ledger that meets or exceeds the configured `online_delete` value. |
```

Comment:
Also maybe note that can_delete returns the ledger sequence when executed. So if somebody wants to know that value without making any changes, they can simply call can_delete with no parameters.